geozero/src/wkt/mod.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) mod conversion {
         }
 
         fn to_ewkt(&self, srid: Option<i32>) -> Result<String> {
-            self.to_wkt_with_opts(WktDialect::Ewkt, CoordDimensions::xyzm(), srid)
```

Comment:
I'm not sure the right way to fix this, but as I understand it, setting `CoordDimensions` says "this output will have these given geometries", and so defaulting to `xyzm` seems incorrect.